collect_app/src/main/res/drawable/start_new_form_button_background.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="@color/color_on_surface_low_emphasis">
```

Comment:
We're following the color pallete of the "Filled Button" here and looking at its [specs](https://m3.material.io/components/buttons/specs#0eea2a85-b4d7-4c74-b08e-98410b9412c7), I think this should really be `colorOnPrimary` with low emphasis (alpha/opacity). We can replicate our implementation in `color_on_surface_low_emphasis.xml`, but use `?colorOnPrimary` in stead to get that color.

Reply:
We already have `color_primary_low_emphasis.xml` so probably this should be used but for the first button (filled one) we should use `colorOnPrimary` see https://m3.material.io/components/buttons/specs#0eea2a85-b4d7-4c74-b08e-98410b9412c7

I've updated it in the last commit: https://github.com/getodk/collect/pull/5531/commits/1f288089b3afa051d724269cd9afb6c31fa0f4b4